pxr/usd/usd/crateFile.cpp

```diff
@@ -3235,6 +3448,16 @@ CrateFile::_ReadStructuralSections(Reader reader, int64_t fileSize)
     if (m.IsClean()) _ReadSpecs(reader);
 }
 
+/* usddoc bootstrap
+ * Every USD file starts with a Bootstrap that tells us information to parse the file.
+ *
+ * Files must start with an identifier (PXR-USDC), the version number and the offset to the Table of Contents.
+ * There is additional reserved space at the end of the bootstrap for future use.
+ *
+ * Version numbers are stored as Major , Minor and Patch followed by unused bytes.
```

Comment:
```suggestion
 * Version numbers are stored as Major, Minor and Patch followed by unused bytes.
```
the table shows 8  bytes for version, how are major, minor, and patch, packed into those 8 bytes?